.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
I just realized that this applies to the entire repo of examples. I don't think it's such a good idea to do that. Can you move it to `eslint-example/.gitignore` ? That will serve the same purpose. That's also a nice place since someone may start by copying the whole folder.